src/pages/Login/styles.js

```diff
@@ -0,0 +1,24 @@
+import { makeStyles } from "@mui/styles";
+
+const useStyles = makeStyles((theme) => ({
```

Comment:
Then if we want to give CSS  then how can we give?

Reply:
> Then if we want to give CSS then how can we give?

If you're only using the given style once, you could use the `sx` prop (https://mui.com/customization/how-to-customize/#use-the-sx-prop).  If it's to be applied everywhere, you'd probably make it part of the theme.  Otherwise, if it's meant to apply to some but not all elements (e.g., when you'd use `class`es), you'll probably want to go with `styled`.  I give an example of this use in the below comment (See: https://mui.com/customization/how-to-customize/#2-reusable-style-overrides).

You could also create a [Higher-order component](https://reactjs.org/docs/higher-order-components.html) to simulate multiple class membership:

```
function withClasses(Component, classes) {
  return classes.reduce((styledComponent, className) => styled(styledComponent)(className), Component);
}

withClasses(Typography, [centerSmLeftNonSmAlignStyle]);
```

Reply:
The style we are giving is for only that particular component, I don't think the same style will work for other components too. So in that case what should we do? Inline CSS using `sx` props?
Like I need to use `paddingTop: 5 and width: 100` 3 times in my component and I'm also supposing I don't have to use them on other components so now I'm using styles.js file, but you're suggesting to use `sx` props in this case, if I understood right?

Reply:
Yes, the underlying principle here is to avoid repetition of values that you always expect to stay the same.  So for example, if `paddingTop` were to change from `5` to `10`, you should only be changing it in one place.

So, in this case, if you have three instances of the same component with the same styles, you could assign the component to a variable as follows (which you'd export/import if you were using across multiple files):
```
const ____ = <Component sx = {{paddingTop: 5, width: 100}}>...</Component>;
```
If on the other hand, these Components had additional differing styles, you could assign an object with the common styles to a variable and then use it as part of the `sx` value as follows:
```
const commonStyles = {paddingTop: 5, width: 100};
<Component sx = {{...commonStyles, color: red}}>...</Component>;
<Component sx = {{...commonStyles, margin: 10}}>...</Component>;
```